plugins/catalog-graph/src/components/EntityRelationsGraph/useEntityRelationGraph.ts

```diff
@@ -68,7 +74,12 @@ export function useEntityRelationGraph({
                   rel.targetRef.startsWith(
                     `${kind.toLocaleLowerCase('en-US')}:`,
                   ),
-                ))
+                )) &&
+              (!ownedBy ||
+                (rel.type === 'ownedBy' &&
+                  ownedBy.includes(rel.targetRef.split('/').slice(-1)[0])) ||
```

Comment:
If you make everything use entity refs as mentioned above, you can use just plain string comparisons here instead of doing parsing and splitting